### ISSUE TITLE
Question about the Trainval_Neg_HICO.pkl

### ISSUE THREAD

**Issue by qqqlz** [DirtyHarryLYL/Transferable-Interactiveness-Network#25]
Thanks for your great work!
I want to know how you generate the Trainval_Neg_HICO.pkl?
And the negative (human verb object) has the incorrect verb?

**Comment by HuangOwen** [comment]
We do not generate this training file by ourselves, please refer to the HICO website.

**Comment by qqqlz** [comment]
Thank you for your reply!
I also have a question that how to calculate the loss for negative samples, for the wrong relationship between the human and object?

**Comment by HuangOwen** [comment]
Positive samples and negative samples differ in the ground truth label. Loss computation is the same (softmax/cross_entropy) between these two.

**Comment by qqqlz** [comment]
Negative samples have wrong relationships. If loss computation is same as positive samples,  minimizing loss does not mean the predictions for these negative samples tend to the wrong relations?

**Comment by HuangOwen** [comment]
Exactly speaking, HOI detection is a multi-label classification problem. 600 HOI classes are separate during the inference (600 sigmoids). So a pair can be pos to one class, and neg to another class. In our paper, we focus on the interactive or non-interactive pair problem. That is to say, whether a pair has one or more HOIs or zero HOI.
